packages/vuetify/src/styles/settings/_variables.scss

```diff
@@ -22,17 +22,18 @@ $rounded: map-deep-merge(
 );
 
 $spacer: 4px !default;
+$spacers-steps-number: 16 !default; 
```

Comment:
```suggestion
$spacers-steps: 16 !default; 
```

Reply:
updated (commit message is wrong by mistake)